packages/components/tailwind.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   /*The main value in TW utility classes is in Storybook stories.
     We avoid using them in component styles to reduce chance of conflict with other libraries.
     Please configure downstream Tailwind config purge to include app files if necessary*/
-  purge: ["./src/**/*.stories.{js,jsx,ts,tsx}"],
+  purge: ["./src/**/*.stories.{js,jsx,ts,tsx}", "./.storybook/*"],
```

Comment:
We talked about this separately but posting here for posterity – I couldn't get any path that didn't specify `.storybook` to work, and @anniehu4 suggested we just point to the whole directory instead of just specifying the `.stories` files.